code/game/machinery/defib_mount.dm

```diff
@@ -46,13 +46,14 @@
 /obj/machinery/defibrillator_mount/examine(mob/user)
 	. = ..()
 	if(defib)
-		. += "<span class='notice'>There is a defib unit hooked up. Alt-click to remove it.<span>"
+		if(!clamps_locked)
+			. += "<span class='notice'>There is a defib unit hooked up. Alt-click to remove it.<span>"
 		if(GLOB.security_level >= SEC_LEVEL_RED)
```

Comment:
Defib mounts did this? What???

Reply:
Yeah, defibs can be stolen by any assistant on red alert (even if they're locked)